Backend/sprintone/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+## Port
+server.port = 5000
+endpoints.env.enabled=true
+
+## MySQL
+spring.datasource.url = jdbc:mysql://localhost:3306/esrs_test
+spring.datasource.username = root
+spring.datasource.password = password
```

Comment:
![NO!](https://media.giphy.com/media/12XMGIWtrHBl5e/giphy.gif)

Many times I've seen this. Many times I've seen databases with default credentials get purged by someone with a script trying random databases for ransom... We need a secure password

Also, do not push these credentials in git at all. Save them as encrypted variables in the Azure Pipeline to replace **only on deploy**. (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch)

Remember, this git repo is public

Reply:
@Xander9 - This shouldn't be here.